File: RQ1_Prompt_Elements/data/scenario2/id_133.java
package scenario2;


import java.util.List;


class SumSquares {
    /**
     * You are given a list of numbers.
     * You need to return the sum of squared numbers in the given list,
     * round each element in the list to the upper int(Ceiling) first.
     */
    public static int sumSquares(List<Number> lst) {
        int sum = 0;
        for (Number n : lst) {
            sum += Math.pow(Math.ceil(n.doubleValue()), 2);
        }
        return sum;
    }

}

File: HumanEvalPython/scenario2/id_133.py


def sum_squares(lst):
    """You are given a list of numbers.
    You need to return the sum of squared numbers in the given list,
    round each element in the list to the upper int(Ceiling) first.
    
    
    

    """

    import math
    squared = 0
    for i in lst:
        squared += math.ceil(i)**2
    return squared

